client/platform/desktop/backend/native/windows.ts

```diff
@@ -46,9 +46,13 @@ const ViameWindowsConstants = {
       '-vf "scale=ceil(iw*sar/2)*2:ceil(ih/2)*2,setsar=1"',
     ].join(' '),
   },
-
 };
 
+function sourceString(settings: Settings) {
+  const setupScriptAbs = npath.join(settings.viamePath, ViameWindowsConstants.setup);
+  return `call "${setupScriptAbs}" >NUL`;
```

Comment:
I can only find references to `call` being used from inside batch files and that it has no effect when called from the cmd line.  Let me know if I am misinterpreting this or if call might be doing something else. It doesn't hurt anything to have it there so you can leave it.

[https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/call](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/call)

Reply:
It makes no difference.  I tested it out to see if, like bash/sh, there was a difference in shell/subshell environment persistence (i.e. if call worked like source) while I was debugging.

It has exactly the same behavior within `cmd.exe`, so it's superfluous.  I can remove it if you'd like.

Reply:
I think take it out because it implies there is the functionality like a batch file that it's there.  If it did act like a batch file  moving the `&&` after the initial setup would work because it would create the call and move onto the next command once the first one is complete or errors out.